.expeditor/config.yml

```diff
@@ -0,0 +1,42 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: sustaining-notify
+
+rubygems:
+  - win32-certstore
+
+github:
+  # The tag format to use (e.g. v1.0.0)
+  version_tag_format: "v{{version}}"
```

Comment:
this isn't the current tag format:
```
$ git tag
win32-taskscheduler-0.2.0
win32-taskscheduler-0.2.1
win32-taskscheduler-0.2.2
win32-taskscheduler-0.3.0
win32-taskscheduler-0.3.1
win32-taskscheduler-0.3.2
win32-taskscheduler-0.4.0
win32-taskscheduler-0.4.1
```

Reply:
nevermind. @Vasu1105 noticed I"m getting ahead of myself.

Reply:
@btm - Here git tag similar to chef that's why we are added "v{{version}}" as:
```
win32-certstore$ git tag
v0.1.0
```
```
chef$ git tag
v12.10.1
v12.10.10
```